### PR TITLE
Bump runtime version

### PR DESCRIPTION
**Pull request by fmuellner**
It's hard to follow Chelsea Manning (and others) when all emojis
show up in black & white, so bump the platform version to the
last stable release that introduced support for color emojis.

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

**Comment by TingPing** [comment]
https://flathub.org/builds/#/builders/1/builds/791

**Comment by baedert** [comment]
@fmuellner Do the color emojis work for you now? They don't here.

**Comment by TingPing** [comment]
@baedert You need the fonts correctly configured on the host, so f27 basically.

**Comment by fmuellner** [comment]
@baedert Yup.

@TingPing Mmmh, is that the case? The 3.26 runtime does include a color emoji font, so I would expect it to work without any involvement of the host.

**Comment by TingPing** [comment]
It uses host configs, so the font being there isn't enough. At least in my testing F26 configs didn't work but F27 configs did. Though maybe something in the runtime could hardcode a default emoji font of emoji-one or something.

**Comment by fmuellner** [comment]
Ah, interesting. I did include fontconfig and friends in the Polari flatpak for a while, and I'm pretty sure I had that working on F26 (always hard to tell exactly with the jhbuild bits thrown in left and right). It did involve some config shenanigans though, maybe the runtime would need some of that as well.
